### PR TITLE
Update names

### LINKED ISSUE #88: Wrong description at Swagger
Missing parameters for swagger documentation at:
- Special prices
- Special Líf Prices

We can see the correct parameters at  `wt-nodejs-api/src/srv/routes/units/prices.js`

### LINKED ISSUE #89: Parameters names
Through the API develop, the route parameter names became less representative. To improve this we must rename those parameters in routes and swagger, 

### Update route parameters
- `hotels/:address` for `hotels/:hotelAddress`
- `units/:unit` for `units/:unitlAddress`
- `hotels/:type` for `unitTypes/:unitType`

### Update Swagger parameters
- `hotels/{address}` for `hotels/{hotelAddress}`
- `units/{unit}` for `units/{unitlAddress}`
- `hotels/{type}` for `unitTypes/{unitType}`

### PR DESCRIPTION
**Pull request by MatiasOS**
This fix #88, fix #89 .
Renames address to hotelAddress and fixs a typo. Renames unit to unitAddress and type to unitType. And fix wrong description with special prices.

### CODE REVIEW

File: docs/swagger.json [hunk 1]
```diff
@@ -1835,6 +1837,14 @@
                 "price": {
                   "description": "Special price.",
                   "type": "number"
+                },
+                "from": {
+                  "description": "Initla date.",
+                  "type": "Date"
+                },
+                "dats": {
```
Comment (JirkaChadima) [inline docs/swagger.json]: This is a typo, right?

File: docs/swagger.json [hunk 2]
```diff
@@ -1894,6 +1906,14 @@
...
```
Comment (JirkaChadima) [inline docs/swagger.json]: Same typo as above

File: docs/swagger.json (hunk 1 above)
Comment (MatiasOS) [reply docs/swagger.json]: Done!

File: docs/swagger.json (hunk 2 above)
Comment (MatiasOS) [reply docs/swagger.json]: Done!

**Pull request merged**